src/components/Hoverable/index.js

```diff
@@ -22,10 +22,25 @@ class Hoverable extends Component {
 
     componentDidMount() {
         document.addEventListener('mousedown', this.resetHoverStateOnOutsideClick);
+
+        // we like to Block the hover on touch devices but we keep it for Hybrid devices so
+        // following logic blocks hover on touch devices.
+        this.touchStartListener = () => {
```

Comment:
```suggestion
        this.disableHover = () => {
```

Reply:
I have to name two event callbacks. Can you suggest both please?